alacritty.yml

```diff
@@ -570,3 +546,27 @@
   #- { key: PageDown,  mods: Shift,   action: ScrollPageDown, mode: ~Alt       }
   #- { key: Home,      mods: Shift,   action: ScrollToTop,    mode: ~Alt       }
   #- { key: End,       mods: Shift,   action: ScrollToBottom, mode: ~Alt       }
+
+#debug:
+  # Display the time it takes to redraw each frame.
+  #render_timer: false
+
+  # Keep the log file after quitting Alacritty.
+  #persistent_logging: false
+
+  # Log level
+  #
+  # Values for `log_level`:
+  #   - None
+  #   - Error
+  #   - Warn
+  #   - Info
+  #   - Debug
+  #   - Trace
+  #log_level: Warn
+
+  # Print all received window events.
+  #print_events: false
+
+  # Record all characters and escape sequences as test data.
+  #ref_test: false
```

Comment:
If you're decided to touch debug, could we remove `ref_test` setting?

Reply:
Why?

Reply:
It doesn't make any sense to have this setting in a config, it will just create random permanent files for you, so making it CLI only makes sense.

Reply:
Can we do this separately? Thinking about it, I really don't want to do anything other than just moving this section around in this PR.

Reply:
Yeah, we can.

P.s.

Don't be so eager, I could be sleeping for a bit.